modules/tinymce/src/core/main/ts/keyboard/KeyboardOverrides.ts

```diff
@@ -35,6 +36,7 @@ const setup = (editor: Editor): Cell<Text> => {
     EnterKey.setup(editor);
     SpaceKey.setup(editor);
     InputKeys.setup(editor);
+    TabKey.setup(editor);
```

Comment:
Should this be here or above? It really depends on whether this should work with RTC or not. If it's not meant to work with RTC then I do wonder if this should be in the model instead (though that's probably a follow up task).

Reply:
This should be safe to enable with RTC. We do leverage tab for indent/outdent in RTC, but that should be compatible with tab for table cells.

Reply:
After further discussion there's a risk around tab attempting to insert a new row at the end of the table. We'll leave this disabled on RTC for the moment, and log a follow up task to make sure tab doesn't break if the 'insert row' command doesn't do anything.

I'd like that new task to also cover table dialogs, I would like them to use commands to make changes so that we can reduce the amount of snooker bundled into the table plugin (ideally it will become zero, but I guess we'll see). In general, our direction for tables is "everything should use commands, and not fail if the command is not implemented".

Reply:
I'll create a Jira so we can follow up on a couple of these things